example/aws/cognito/README.md

```diff
@@ -0,0 +1,270 @@
+# Deploying Kubeflow with AWS Cognito as idP
+
+This guide describes how to deploy Kubeflow on AWS EKS using Cognito as identity provider.
+
+## Pre-requisites
+
+1. EKS cluster
+2. Kustomize 3.2.x needed for Kubeflow 1.3. [More Info](https://github.com/kubeflow/manifests/tree/v1.3-branch#prerequisites)
+    1. ```
+        curl -sLO https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv3.2.1/kustomize_kustomize.v3.2.1_linux_amd64
```

Comment:
I would say we should remove this ....few people might be trying out from linux or some other machine too. 

Just adding a prerequisite name `kustomize` should be enough.

Reply:
will link to pre-requisites in aws directory once #6 is merged